Procfile

```diff
@@ -1,2 +1 @@
-web: cd client && npm start
-api: bundle exec rails s -p 3001
+web: bundle exec rails s -p $PORT
```

Comment:
Hmm. We've been using foreman to bundle these commands so that we start both servers with `rake start`. Did you set $PORT in Heroku so that it starts the rails server on 3001 _and_ our webpack server on 3000? To me this looks just like the rails server, but maybe you set up webpack on Heroku? 

Anyways, I'm just not sure how to configure this so that it works both locally and on Heroku.

Reply:
Okay just fixed this in the last commit, so we're good to go here :) thanks for your help @wdoug !